translations/es.ts

```diff
@@ -1508,7 +1508,7 @@ Compártelo con tus amigos para poder comunicarte.</translation>
     <message>
         <location filename="../src/widget/form/profileform.ui" line="125"/>
         <source>This QR code contains your Tox ID. You may share this with your friends as well.</source>
-        <translation>Este código QR contiene tu Tox ID. Puedes compartirla con tus amigos.</translation>
+        <translation>Este código QR contiene tu Tox ID. Puedes compartirlo con tus amigos.</translation>
```

Comment:
No, you'R talking about the QR code which you cna share with your friends. (You may share this with your friends as well, instead of sending the id string in an e-mail or copying by hand ...)